src/DynamoCoreWpf/ViewModels/PackageManager/PackageViewModel.cs

```diff
@@ -45,7 +45,6 @@ public bool HasAssemblies
             get { return Model.LoadedAssemblies.Any(); }
         }
 
-        public DelegateCommand ToggleTypesVisibleInManagerCommand { get; set; }
```

Comment:
I guess this should be made a noop as well?

Reply:
Yep, I was going back and forth about that (again). But I guess we should do the same for every command, regardless how unlikely it is that a 3rd party have used it.